### PR TITLE
fixes bug 1250142 - Read from .json instead of _info.txt where possible

### PR DESCRIPTION
**Pull request by peterbe**
cc @KaiRo-at 

Bye-bye depending on `*_info.txt` files. Now it looks for `.json` files instead. 

When I run this locally, it finds both nightly, release and beta for firefox. That hasn't changed. 

The scraping of `*_info.txt` files was only applicable to mobile (aka fennec), seamonkey and thunderbird. These now HAVE `.json` files with all the juicy stuff in them. They didn't use to have that.

### CODE REVIEW

**Comment by peterbe** [comment]
When I run this locally like this:
```
python socorro/cron/jobs/ftpscraper.py --product=mobile --date=2016-02-21
```
It writes on `stdout` that there are nightly, beta and release releases.

**Comment by peterbe** [comment]
Re-opening so travis attempts this PR.

File: socorro/cron/jobs/ftpscraper.py [hunk 1]
```diff
@@ -136,6 +136,7 @@ def parse_build_json_file(self, url, nightly=False):
                 )
 
     def parse_info_file(self, url, nightly=False):
+        self.config.logger.debug('Opening %s', url)
```
Comment (adngdb) [inline socorro/cron/jobs/ftpscraper.py]: What were you saying about excessive logging? ;-)

**Comment by adngdb** [comment]
r+

**Pull request merged**